File: Lesson 1/Example/One/main.py
from book import Book


def main():
    book1 = Book("harry Potter", 1997, 35.99, True, '1')
    book2 = Book("lord of the rings", 1954, 19.99, True, '2')

    print(book1.title.upper())
    print(book2.title)


if __name__=="__main__":
    main()

File: Lesson 1/Example/One/Main.java
public class Main
{
    public static void main(final String[] args)
    {
        Book book1 = new Book("harry potter", 1997, 35.99, true, '1');
        Book book2 = new Book("lord of the rings", 1954, 19.99, true, '2');

        System.out.println(book1.getTitle().toUpperCase());
        System.out.println(book2.getTitle());
    }
}